sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -27,6 +27,9 @@
 
 final class TracerProviderConfiguration {
 
+  private static final double DEFAULT_SAMPLE_RATIO = 1.0d;
```

Comment:
```suggestion
  private static final double DEFAULT_TRACEIDRATIO_SAMPLE_RATIO = 1.0d;
```

Stuttering isn't great, but without the sampler type here the variable seems mysterious